# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264196047 18000
#      Fri Jan 22 16:34:07 2010 -0500
# Node ID 30aeda033364fef4405d01d55cc9a01ecbbefc47
# Parent  8fa6ce1b21f236ff90403498e00acb97f9ed77cd
octave.cc: don't include sys/types.h, unistd.h, or fstream

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-22  John W. Eaton  <jwe@octave.org>
+
+	* octave.cc: Don't include sys/types.h, unistd.h, or fstream.
+
 2010-01-22  John W. Eaton  <jwe@octave.org>
 
 	* oct-parse.yy (make_break_command, make_continue_command,
 	make_return_command): Don't examine evaluator state here.
 	* pt-eval.cc (tree_evaluator::visit_break_command): Don't set
 	tree_break_command::breaking unless inside function or script
 	body, or inside a looping command.
 	(tree_evaluator::visit_continue_command): Likewise, for
diff --git a/src/octave.cc b/src/octave.cc
--- a/src/octave.cc
+++ b/src/octave.cc
@@ -27,22 +27,18 @@ along with Octave; see the file COPYING.
 #include <config.h>
 #endif
 
 #include <cassert>
 #include <cstdlib>
 #include <cstring>
 #include <ctime>
 
-#include <fstream>
 #include <iostream>
 
-#include <sys/types.h>
-#include <unistd.h>
-
 #include "cmd-edit.h"
 #include "f77-fcn.h"
 #include "file-ops.h"
 #include "file-stat.h"
 #include "lo-error.h"
 #include "oct-env.h"
 #include "pathsearch.h"
 #include "str-vec.h"
